test/fixtures/render/enum-default.json

```diff
@@ -199,7 +199,7 @@
                                   "content": "application/json"
                                 }
                               },
-                              "content": "{\n  \"enumField\": \"c\"\n}"
+                              "content": "{\n  \"enumField\": \"a\"\n}"
```

Comment:
I think this is a regression. Before we was using the default value `c` and now we're using the first enumeration value. I believe the default should take precedence.